lib/graphql_devise/rails/routes.rb

```diff
@@ -14,7 +14,8 @@ def mount_graphql_devise_for(resource, opts = {})
         logout:              GraphqlDevise::Mutations::Logout,
         sign_up:             GraphqlDevise::Mutations::SignUp,
         update_password:     GraphqlDevise::Mutations::UpdatePassword,
-        send_password_reset: GraphqlDevise::Mutations::SendPasswordReset
+        send_reset_password: GraphqlDevise::Mutations::SendPasswordReset,
+        resend_confirmation: GraphqlDevise::Mutations::ResendConfirmation
       }.freeze
       default_queries = {
         confirm_account:      GraphqlDevise::Resolvers::ConfirmAccount,
```

Comment:
I was debating whether or not to make `ResendConfirmation` a mutation or a query. Technically, it doesn't change the underlying data but it does send an email which in a way is a kind of mutation.